target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.c

```diff
@@ -0,0 +1,461 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/* Realtek RTL83XX clock driver
```

Comment:
ah wait, there is a real clock driver for mips; but I suppose you it's not fair to remove the mach specific stuff to be replaced with this as part of this patch series?